english-chinese-translator/eng-chinese-translator.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
```

Comment:
The UI looks good. But the intention is to let a API Gateway URL to show this page. 
Ultimate goal for you is that when you run nestjs app locally, i.e
```
npm run start
```
You should be able to visit this UI at:
http://localhost:3000/eng-chinese-translator